staging/src/k8s.io/csi-translation-lib/plugins/vsphere_volume_test.go

```diff
@@ -264,6 +352,65 @@ func TestTranslateVSphereInTreePVToCSI(t *testing.T) {
 			},
 			expErr: false,
 		},
+		{
+			name: "translate valid vSphere in-tree PV with topology labels to vSphere CSI PV",
+			intreePV: &v1.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:   "pvc-d8b4475f-2c47-486e-9b57-43ae006f9b59",
+					Labels: map[string]string{"failure-domain.beta.kubernetes.io/zone": "z1", "failure-domain.beta.kubernetes.io/region": "r1"},
```

Comment:
Should we have another test for In-tree PV with GA labels?

Reply:
yes